src/visu_config.xsd

```diff
@@ -1118,11 +1208,31 @@
       <xsd:element name="layout" type="layout" minOccurs="0" maxOccurs="1"/>
       <xsd:element name="label" type="label" minOccurs="0" maxOccurs="1"/>
     </xsd:sequence>
-    <xsd:attribute name="src" type="uri" use="required" />
-    <xsd:attribute name="width" type="dimension" use="optional" />
-    <xsd:attribute name="height" type="dimension" use="optional" />
+    <xsd:attribute name="src" type="uri" use="required">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">Complete path (URI) to the picture</xsd:documentation>
+        <xsd:documentation xml:lang="de">vollständiger Pfad zum Bild (URI)</xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="width" type="dimension" use="optional">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">Sets the width of the image in pixels</xsd:documentation>
```

Comment:
No, `width` and `height` are in "CSS units". So when the user wants pixels it must be in the format `123px`. But it could also be e.g. `12.3em`, `1.23%`, ...